### PR TITLE
Refactor/accessibilities added

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [ ] Feature
- [x] Refactor
 
Describe the feature - fix - refactor: refactor for accessibilities purposes

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test: styles.css

Additional comments:

### CODE REVIEW

**Review by lschnell8 — APPROVED** [review]
This is awesome! Thanks for tweaking things for accessibility and the page looks great with the new CSS!

File: src/styles.css [hunk 1]
```diff
@@ -1,27 +1,40 @@
 /* @import './node_modules/normalize.css/normalize.css'; */
 
 * {
-  font-family: 'Open sans', sans-serif;
+  font-family: 'Mochiy Pop P One', sans-serif;
+  font-size: 16px;
+  border-radius: 5px;
 }
 
+$primary-color: #396204;
```
Comment (StevieAmb) [inline src/styles.css:9]: Yes, SASS!!!

File: src/styles.css [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
+  background-attachment: fixed;
 }
 
 header {
   display: flex;
   justify-content: space-between;
   height: 15vh;
   align-items: center;
-  background-color: goldenrod;
+  background-color: $primary-color;
```
Comment (StevieAmb) [inline src/styles.css:29]: Look at these variables!

File: src/styles.css [hunk 3]
```diff
@@ -57,9 +70,15 @@ input {
 .unfavorite-button,
 .favorite-button {
   align-self: flex-end;
-  height: 5vh;
+  height: 12vh;
   width: 10vw;
-  border-radius: 3px;
+}
+
+.favorite-button:hover,
```
Comment (StevieAmb) [inline src/styles.css:77]: Gorgeous hover states!

File: src/styles.css [hunk 4]
```diff
@@ -134,18 +167,32 @@ ol {
...
   border: none;
   cursor: pointer;
-}
-
-.dropbtn:hover, .dropbtn:focus {
-  background-color: #2980B9;
+  /* border-radius: 8px; */
+}
+
+.dropbtn:hover,
```
Comment (StevieAmb) [inline src/styles.css:185]: Down for the focus!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**